src/rest/queue.ts

```diff
@@ -62,7 +62,7 @@ export async function processQueue(id: string) {
         response.headers,
       );
 
-      if (response.status < 200 && response.status >= 400) {
+      if (response.status < 200 || response.status >= 400) {
```

Comment:
I like this more if you have nothing against changing it.
```suggestion
      if (response.status < 200 || 400 <= response.status) {
```